spec.bs

```diff
@@ -228,27 +230,35 @@ The {{SharedStorageWorklet}} object allows developers to supply [=module scripts
   };
 </xmp>
 
-Each {{SharedStorageWorklet}} has an associated boolean <dfn>module added</dfn>, initialized to false.
+Each {{SharedStorageWorklet}} has an associated boolean <dfn>addModule initiated</dfn>, initialized to false.
```

Comment:
Later in the spec you link to `[=boolean=]` for associated booleans, but not here. I'd probably try and be consistent (bias towards always linking).